src/Native/build-native.proj

```diff
@@ -0,0 +1,68 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+
+  <!-- Target that builds all the native binaries in the Native folder -->
+  <Target Name="Build" DependsOnTargets="BuildNativeUnix;BuildNativeWindows" />
+
+  <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
+    <GenerateVersionSourceFile>true</GenerateVersionSourceFile>
+    <NativeVersionSourceFile>$(BaseIntermediateOutputPath)version.c</NativeVersionSourceFile>
+  </PropertyGroup>
+
+  <PropertyGroup  Condition="'$(OS)' == 'Windows_NT'">
+    <GenerateNativeVersionInfo>true</GenerateNativeVersionInfo>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!--
+      TODO: Need to add support for forwarding all parameters config.json supported.
```

Comment:
:+1:  Agreed.  Especially the `_StripSymbolsArg` below - we need to be able to turn this on and off.